docs/book/src/plugins/declarative-v1.md

```diff
@@ -0,0 +1,37 @@
+# Declarative Plugin
```

Comment:
We need also to add a new section in:

<img width="287" alt="Screenshot 2022-03-26 at 09 43 35" src="https://user-images.githubusercontent.com/7708031/160233990-60d4d4b9-ccf9-46bf-b9ec-2c3966465094.png">

Maybe 
`7. Available Plugins`

With this subsection:

7.1  Declarative Plugin

WDYT?

Reply:
See: https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/SUMMARY.md

Reply:
@mariuskimmina, 

> also, should it be 7. Available Plugins or 6.3 Available Plugins? I think I like to keep them in the general Plugins section

Yes, that makes sense 👍

Reply:
I have added an available plugins section now